tests/shared/src/test/scala/scala/meta/tests/parsers/BasicPositionSuite.scala

```diff
@@ -17,6 +17,14 @@ class BasicPositionSuite extends BasePositionSuite(dialects.Scala213) {
     "a f (b)"
   )
 
+  checkPositions[Term](
+    "a f (123)"
+  )
+
+  checkPositions[Term](
+    "a f ()"
+  )
```

Comment:
this was not failing before, but I thought I could add it preemptively since there is clearly a special case for Unit